ssdutil/main.py

```diff
@@ -16,11 +17,20 @@
 
 DEFAULT_DEVICE="/dev/sda"
 SYSLOG_IDENTIFIER = "ssdutil"
+DISK_TYPE_SSD = "0"
 
 # Global logger instance
 log = logger.Logger(SYSLOG_IDENTIFIER)
 
 
+def get_disk_type(diskdev):
```

Comment:
better to validate if the listed diskdev is valid as listed by "lsblk" otherwise the utility will crash. you can use 
```
lsblk -l -n | grep 'disk' 
```
to get the list of all disk names available in the host